feature-libs/product-multi-dimensional/selector/components/selector/product-multi-dimensional-selector.component.html

```diff
@@ -21,7 +21,17 @@
               (click)="changeVariant(option.code)"
               [cxFocus]="{ key: option.code }"
             >
-              <cx-media [container]="option.image" role="img" />
+              <cx-media
+                *cxFeature="'useExtendedMediaComponentConfiguration'"
+                [container]="option.image"
+                role="img"
+                [elementType]="'img'"
+              />
+              <cx-media
+                *cxFeature="'!useExtendedMediaComponentConfiguration'"
+                [container]="option.image"
+                role="img"
+              />
```

Comment:
`query:` this usage of `cx-media` doesn't contain `format` property. Is the feature toggle wrapper required?